--- /src/main/java/introclassJava/median_48b82975_000.java
+++ /src/main/java/introclassJava/median_48b82975_000.java
@@ -69,9 +69,9 @@
-		}
-		if (((b.value <= a.value) && (a.value <= c.value)) || 
-		((c.value <= a.value) && (a.value <= b.value))) {
-			output += java.lang.String.format("%d is the median\n", a.value);
-		} else {
-			output += java.lang.String.format("%d is the median\n", c.value);
-		}
-		if (true) 
-		{ 			return;}
+
+
+
+
+
+
+
+
+			return;} 		if (((b.value <= a.value) && (a.value <= c.value)) || ((c.value <= a.value) && (a.value <= b.value))) { 			output += java.lang.String.format("%d is the median\n", a.value);} else { 			output += java.lang.String.format("%d is the median\n", c.value);} 		if (true) { 			return;}

